.github/workflows/build.yml

```
@@ -0,0 +1,19 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)